modules/common-workflows/pages/workflow-reducing-sync-times-over-wan.adoc

```diff
@@ -0,0 +1,120 @@
+[[workflow-reducing-sync-times-over-WAN]]
+
+= Reducing high sync times between {productname} Server and Proxy over WAN connections
+
+Depending on what changes are executed in the WebUI or via an API call to distribution or system settings, [path]``cobbler sync`` command is required to transfer files from {productname} server to {productname} Proxy systems.
+To accomplish this, cobbler uses a list of proxies specified in [path]``/etc/cobbler/settings``.
+
+
+== Use case
+
+Due to its design, [path]``cobbler sync`` is not able to sync only the changed or recently added files.
+
+Instead, executing [path]``cobbler sync`` triggers a full sync of the [path]``/srv/tftpboot`` directory to all specified proxies configured in [path]``/etc/cobbler/settings``.
+It is also influenced by the latency of the WAN connection between the involved systems.
+
+The process of syncing may take a considerable amount of time to finish according to the logs in [path]`` /var/log/cobbler/``.
+
+In one example, it started at:
+
+----
+Thu Jun  3 14:47:35 2021 - DEBUG | running python triggers from /var/lib/cobbler/triggers/task/sync/pre/*
+Thu Jun  3 14:47:35 2021 - DEBUG | running shell triggers from /var/lib/cobbler/triggers/task/sync/pre/*
+----
+
+and ended at:
+
+----
+Thu Jun  3 15:18:49 2021 - DEBUG | running shell triggers from /var/lib/cobbler/triggers/task/sync/post/*
+Thu Jun  3 15:18:49 2021 - DEBUG | shell triggers finished successfully
+----
+
+The transfer amount was roughly 1.8GB.
+The transfer took almost 30 minutes.
+
+By comparison, copying a single big file of the same size as [path]``/srv/tftboot`` completes  within several minutes.
+
+
+
+== Resolution
+
+Switching to an [path]``rsync``-based approach to copy files between {productname} Server and Proxy may help to reduce the transfer and wait times. 
+
+To accomplish this task, a script has been developed.
+It is available for download at <https://suse.my.salesforce.com/sfc/p/1i000000gLOd/a/1i000000ll5B/B2AmvIJN2_JsAyjTQzCVP_x5ioVgd0bYN9X9NpMugS8>.
+
+[WARNING]
+====
+There is no support available for individual adjustments of the script.
+ 
+The script and the comments inside aim to provide a good overview of the process and steps to be taken into consideration. 
+If further help is required, contact SUSE Consulting.
+====
+
+
+== Preparation tasks
+
+The proposed approach is benefitial in the following environment:
+. SUSE Manager Proxy systems are connected via a WAN connection;
+. [path]``/srv/tftboot`` contains a high number of files for distributions and client PXE boot files, in total several thousand files;
+. Any proxy in [path]``/etc/cobbler/settings`` has been disabled, otherwise {productname} will continue to sync content to the proxies.
```

Comment:
```suggestion
. Any proxy in [path]``/etc/cobbler/settings.yaml`` has been disabled, otherwise {productname} will continue to sync content to the proxies.
```